_format/1.1/index.md

```diff
@@ -530,13 +530,22 @@ of this member **MUST** be an object (a "links object").
 <a href="#document-links-link" id="document-links-link"></a>
 Within this object, a link **MUST** be represented as either:
 
-* a string containing the link's URI.
+* a URI-reference [[RFC3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1)] to the link's target.
 * <a id="document-links-link-object"></a>an object ("link object") which can
   contain the following members:
-  * `href`: a string containing the link's URI.
+  * `href`: a URI-reference to the link's target.
```

Comment:
This is the only place that URI-reference is mentioned without also including a link to RFC3986 Section 4.1

Reply:
Fixed.